rfc5424logging/handler.py

```diff
@@ -174,6 +175,13 @@ def emit(self, record):
         The record is formatted, and then sent to the syslog server. If
         exception information is present, it is NOT sent to the server.
         """
+        try:
+            syslog_msg = self.construct_rfc5424_message(record)
+            self.send_to_socket(syslog_msg)
+        except Exception:
+            self.handleError(record)
```

Comment:
These changes were just to make it a little easier to subclass and modify specific behavior so the concerns of making the message and sending it to the socket were not entangled.